app/grandchallenge/cases/urls.py

```diff
@@ -30,4 +32,14 @@
         VTKImageDetail.as_view(),
         name="vtk-image-detail",
     ),
+    path(
+        "images/<uuid:pk>/cs/",
+        CSImageDetail.as_view(),
+        name="cs-image-detail",
+    ),
+    path(
+        "images/<uuid:pk>/cs/loader",
```

Comment:
```suggestion
        "images/<uuid:pk>/cs/loader/",
```

Missing trailing slash